spk/bazarr/src/service-setup.sh

```diff
@@ -1,9 +1,7 @@
-FFMPEG_DIR="/var/packages/ffmpeg/target"
-PYTHON_DIR="/var/packages/python38/target"
-PATH="${SYNOPKG_PKGDEST}/bin:${SYNOPKG_PKGDEST}/env/bin:${PYTHON_DIR}/bin:${FFMPEG_DIR}/bin:${PATH}"
+FFMPEG_DIR="/var/packages/ffmpeg/target/bin"
```

Comment:
Though it does work I tend to prefer using explicit variable names, here the "home" of FFMPEG is technically  "/var/packages/ffmpeg/target", "bin" is the directory you would be putting in the PATH. If, for example there was a "lib" directory to include for library lookup you would be reusing the FFMPEG_DIR.

But that's not critical, I give you that.